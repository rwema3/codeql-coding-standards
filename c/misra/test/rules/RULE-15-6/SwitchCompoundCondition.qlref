rules/RULE-15-6/SwitchCompoundCondition.ql