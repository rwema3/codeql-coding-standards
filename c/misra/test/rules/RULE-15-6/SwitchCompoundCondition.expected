| test.c:75:3:79:5 | switch (...) ...  | Switch body not enclosed within braces. |
